src/components/MessageList.vue

```diff
@@ -79,6 +79,7 @@
 <script>
 'kiwi public';
 
+import state from '@/libs/state';
```

Comment:
Components can get access to the state via `this.$state` instead of importing it in